apps/alert_processor/lib/model/subscription.ex

```diff
@@ -446,8 +448,16 @@ defmodule AlertProcessor.Model.Subscription do
          [route_ids: _, routes: _, stops: _, wildcard: true],
          alert
        ) do
-    from(s in __MODULE__, select: s.id)
-    |> where_not_paused_and_not_yet_notified(alert)
+    start_time = System.monotonic_time(:millisecond)
+
+    subscription_ids =
+      from(s in __MODULE__, select: s.id)
+      |> where_not_paused_and_not_yet_notified(alert)
+
+    end_time = System.monotonic_time(:millisecond)
+    Logger.info("wildcard query, time=#{end_time - start_time}")
```

Comment:
suggestion(non-blocking): this looks like something you could wrap up into a function (easier) or a macro (more efficient)
```elixir
def log_timing(key, fun) when is_function(fun, 0) do
  start_time = System.monotonic_time(:millisecond)
  result = fun.()
  end_time = System.monotonic_time(:millisecond)
  Logger.info("#{key} time=#{end_time - start_time}")
  result
end

log_timing("wildcard query", fn ->
  from(s in __MODULE__, select: s.id)
  |> where_not_paused_and_not_yet_notified(alert)
end)